css/templates/travel/components/_results-navbar.css

```diff
@@ -0,0 +1,18 @@
+.travel-results-navbar {
+  margin-top: -4rem;
+  margin-right: -4rem;
+  margin-left: -4rem;
+  background-color: var(--purple);
+  color: var(--white);
+  animation: fade-and-slideup 0.4s ease-in-out both;
+}
+
+.travel-results-navbar .line-height-1 {
+  line-height: 1;
+}
+
+@media (--breakpoint-md) {
+  .travel-results-navbar .line-height-1 {
```

Comment:
i kind of hate this model , can you give another class name to the target and make the line height inherit on that class name instead.

Reply:
makes sense, done